db/migrate/20250814153241_create_mhv_metrics_unique_user_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMHVMetricsUniqueUserEvents < ActiveRecord::Migration[7.2]
+  def change
+    create_table :mhv_metrics_unique_user_events, if_not_exists: true, id: false do |t|
+      # Note: user_id intentionally has NO foreign key constraint by design
+      # Benefits: Performance optimization for high-volume inserts, historical data preservation
+      # even if users are deleted, and operational simplicity for this analytics table
+      t.uuid :user_id, null: false, comment: 'Unique user identifier'
+      t.string :event_name, limit: 50, null: false, comment: 'Event type name'
```

Comment:
The default character limit for string type in postgres is 255 characters. I'd recommend using the default unless you have a good reason to change this.

Reply:
The intention is to keep the table small, so limiting to 50 characters is more for developers not to store a whole paragraph as an event name. I wanted to put a constraint on it. Why do you recommend keeping it as the default? I don't see any performance advantage to not specifying the limit.

Reply:
Ah, I was incorrect with my statement on a `string` type yielding a `VARCHAR(255)` column type. It ends up being a `VARCHAR` for later versions of Rails/ActiveRecord.

To me, 50 characters doesn't seem very long. Without example data its possible that input could be truncated.